src/store.js

```diff
@@ -0,0 +1,18 @@
+import { createStore, applyMiddleware, compose } from 'redux';
+import thunk from 'redux-thunk';
+import rootReducer from './reducers';
+
+const initialState = {};
+
+const middleware = [thunk];
+
+const store = createStore(
+    rootReducer, 
+    initialState,
+    compose( 
+        applyMiddleware(...middleware),
+        window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
```

Comment:
here `compose` or alternative can be used, these are basically from **_Redux-Chrome Add-on_**, provided by `devTools`.
`REDUX_DEVTOOLS_EXTENSION` this is the syntax provided by the DevTools add on.